Frontend/kalvians_management_system-kms/src/Routes/AllRoutes.jsx

```diff
@@ -14,7 +14,7 @@ import Tickets from '../Pages/Tickets';
 const AllRoutes = () => {
   return (
     <Routes>
-        <Route path="/" element={<Home></Home> }></Route>
+        <Route path="/dashboard" element={<Home></Home> }></Route>

```

Comment:
No need to write `<Home></Home>` you can use `<Home/>` change for all the components.